tests/qasm3/test_if.py

```diff
@@ -136,6 +129,79 @@ def test_complex_if():
     check_unrolled_qasm(dumps(result), expected_qasm)
 
 
+def test_multi_bit_if():
+    qasm = """OPENQASM 3.0;
+    include "stdgates.inc";
+    qubit[1] q;
+    bit[4] c;
+    if(c == 3){
+        h q[0];
+    }
+    if(c >= 3){
+        h q[0];
+    } else {
+        x q[0];
+    }
+    if(c <= 3){
+        h q[0];
+    } else {
+        x q[0];
+    }
+    if(c < 4){
+        h q[0];
+    } else {
+        x q[0];
+    }
+    """
+    expected_qasm = """OPENQASM 3.0;
+    include "stdgates.inc";
+    qubit[1] q;
+    bit[4] c;
+    if (c[3] == false) {
```

Comment:
I think using the big-endian notation will be more natural here because we mostly define a bit register like `bit[4] c = "0011";`. In this case, `c[0]` will be `false`, `c[1]` as `false`, so on.

Reply:
```python
In [6]: bin(18)[2:].zfill(8)
Out[6]: '00010010'
```
In Python too, the representation is big-endian

Reply:
Got it, will fix